samples/boards/hexiwear/blinky/README.rst

```diff
@@ -0,0 +1,31 @@
+.. _blinky-sample:
```

Comment:
document labels must be unique across the entire documentation set so give this a unique name (maybe hexiwear-blinky-sample)? . (There's already a blinky-sample in samples/basic/blinky/README.rst)